src/workerd/io/BUILD.bazel

```diff
@@ -58,6 +58,7 @@ wd_cc_library(
         "//src/workerd/util:perfetto",
         "@capnp-cpp//src/kj/compat:kj-brotli",
         "@capnp-cpp//src/kj/compat:kj-gzip",
+        # TODO: Remove simdutf once api/crypto has it's own BUILD.bazel
```

Comment:
Nit: I feel hesitant about even adding a TODO here – I did try to modularizing the io target before, but achieving this will be lots of work based on the large amount of interdependencies we have currently. When we're doing this, it will enable much more cleanup than dropping simdutf here.